Shared/Models/Responses/SsmlOutputSpeech.cs

```diff
@@ -1,4 +1,4 @@
-using System.ComponentModel.DataAnnotations;
+using Newtonsoft.Json;
 
 namespace Slight.Alexa.Framework.Models.Responses
 {
```

Comment:
In the main version this class is public. With this SsmlSpeechOutput is inaccessible.